cardinal/world.go

```diff
@@ -182,7 +181,7 @@ func NewWorld(opts ...WorldOption) (*World, error) {
 			)
 		}
 	}
-
+	world.QueryManager.BuildQueryIndex(world)
```

Comment:
See my comment on the query manager, I don't think a query index is necessary and just makes things more clunky because you can easily forget to rebuild the query index.

Reply:
removed.